pkgs/development/python-modules/pgpy/default.nix

```diff
@@ -24,24 +25,30 @@ buildPythonPackage rec {
     hash = "sha256-47YiHNxmjyCOYHHUV3Zyhs3Att9HZtCXYfbN34ooTxU=";
   };
 
-  nativeBuildInputs = [ setuptools ];
+  build-system = [ setuptools ];
 
-  propagatedBuildInputs = [
+  dependencies = [
     pyasn1
+    standard-imghdr
```

Comment:
can't find any references why this is necessary

Reply:
ah yes i see, should we make it conditional, only for python 3.13 ? otherwise I am not sure what copy of imghdr will get loaded for 3.12